tests/iam_policies/user-role.cfn.yaml

```diff
@@ -739,7 +739,7 @@ Resources:
               - logs:CreateLogStream
               - logs:PutLogEvents
             Effect: Allow
-            Resource: !Sub arn:${AWS::Partition}:logs:*:*:*
+            Resource: !Sub arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/parallelcluster/*
```

Comment:
This will break logging for lambda and any other resource that does not write to `/aws/parallelcluster/*` because if I recall correctly the boundary is applied to all cluster roles when specified

Reply:
Fixed: https://github.com/aws/aws-parallelcluster/pull/3678/commits/5dd7e72ee4b543375394ea2ac2d2d281ffa30eff